addons/hw_posbox_homepage/controllers/homepage.py

```diff
@@ -0,0 +1,467 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+import subprocess
+import threading
+import logging
+import platform
+import jinja2
+import os
+import sys
+
+from pathlib import Path
+from odoo import http, tools
+from odoo.addons.hw_drivers.tools import helpers
+from odoo.addons.hw_drivers.main import iot_devices
+from odoo.addons.web.controllers.home import Home
+from odoo.addons.hw_drivers.connection_manager import connection_manager
+from odoo.tools.misc import file_path
+from odoo.addons.hw_drivers.server_logger import (
+    check_and_update_odoo_config_log_to_server_option,
+    get_odoo_config_log_to_server_option,
+    close_server_log_sender_handler,
+)
+
+_logger = logging.getLogger(__name__)
+
+IOT_LOGGING_PREFIX = 'iot-logging-'
+INTERFACE_PREFIX = 'interface-'
+DRIVER_PREFIX = 'driver-'
+AVAILABLE_LOG_LEVELS = ('debug', 'info', 'warning', 'error')
+AVAILABLE_LOG_LEVELS_WITH_PARENT = AVAILABLE_LOG_LEVELS + ('parent',)
+
+if hasattr(sys, 'frozen'):
+    # When running on compiled windows binary, we don't have access to package loader.
+    path = os.path.realpath(os.path.join(os.path.dirname(__file__), '..', 'views'))
+    loader = jinja2.FileSystemLoader(path)
+else:
+    loader = jinja2.PackageLoader('odoo.addons.hw_posbox_homepage', "views")
+
+jinja_env = jinja2.Environment(loader=loader, autoescape=True)
+jinja_env.filters["json"] = json.dumps
+
+index_template = jinja_env.get_template('index.html')
+logs_template = jinja_env.get_template('logs.html')
```

Comment:
I think it could be interesting to get completely rid of jinja as there is no more added value to use it. The 2 views loaded here are just plain HTML files.

We can think of just reading the plain HTML file with `odoo.tools.misc.file_open,` or use `odoo.addons.base.models.ir_qweb.render`like in the web database manager view:
https://github.com/odoo/odoo/blob/b63b830174c99fa0df659978c3f492452eeab3ec/addons/web/controllers/database.py#L57

Reply:
Next imp, i didn't have any time for that before oxp